spec/demo_mode/cli_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'sorbet-runtime'
 require 'demo_mode/cli'
```

Comment:
This change reproduces the issue. See https://github.com/Shopify/cli-ui/blob/v2.3.0/lib/cli/ui.rb#L4-L6